tracing/zipkin/collector.go

```diff
@@ -6,18 +6,33 @@ import "strings"
 // remote endpoints.
 type Collector interface {
 	Collect(*Span) error
+	ShouldSample(*Span) bool
 	Close() error
 }
 
 // NopCollector implements Collector but performs no work.
-type NopCollector struct{}
+type NopCollector struct {
+	shouldSample Sampler
```

Comment:
yes... I first thought I needed it for testing but found out writing the tests I didn't. so it can be removed...